src/Raven.Server/NotificationCenter/Notifications/Details/ConflictPerformanceDetails.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Sparrow.Json.Parsing;
+using static Raven.Server.NotificationCenter.Revisions;
+
+namespace Raven.Server.NotificationCenter.Notifications.Details
+{
+    public class ConflictPerformanceDetails : INotificationDetails
+    {
+        public ConflictPerformanceDetails()
+        {
+            Details = new Dictionary<string, ActionDetails>(StringComparer.OrdinalIgnoreCase);
+        }
+
+        public Dictionary<string, ActionDetails> Details { get; set; }
+
+        public DynamicJsonValue ToJson()
+        {
+            var djv = new DynamicJsonValue();
+            foreach (var (key, details) in Details)
+            {
+                djv[key] = new DynamicJsonValue
+                {
+                    [nameof(ActionDetails.Reason)] = details.Reason,
+                    [nameof(ActionDetails.Deleted)] = details.Deleted,
+                    [nameof(ActionDetails.Time)] = details.Time
+                };
+            }
+
+            return new DynamicJsonValue(GetType())
+            {
+                [nameof(Details)] = djv
+            };
+        }
+
+        public void Update(ConflictInfo info)
+        {
+            Details[info.Id] = new ActionDetails { Reason = info.Reason.ToString(), Deleted = info.Deleted, Time = info.Time };
```

Comment:
keep a Queue here with last 64

Reply:
Done

Reply:
The `Revisions.cs` is keeping a queue of the last 64 that can be now with duplications because it is a simple ConcurrentDictionary (more than 1 info about the same doc).
Here we have a dictionary, so we don't have info about the same doc more than one..
(We keep in the dictionary the **last** exceeding info for every doc).

Reply:
Yes, I'm aware, Queue is enough, check how this is done in PagingPerformanceDetails

Reply:
Done